src/gax/src/path_parameter.rs

```diff
@@ -35,10 +36,15 @@ where
     }
 }
 
-#[derive(thiserror::Error, Debug)]
-pub enum Error {
-    #[error("missing required parameter {0}")]
-    MissingRequiredParameter(String),
+pub mod errors {
+    use crate::error::Error;
+
+    pub fn missing(name: &str) -> Error {
```

Comment:
I think we explicitly exclude the `path_parameter` module from the documentation, but I would still write a Rustdoc comment.